package.json

```diff
@@ -38,6 +38,7 @@
 		"react-apollo": "^2.5.8",
 		"react-datepicker": "^2.9.6",
 		"react-dom": "^16.8.6",
+		"react-html-parser": "^2.0.2",
```

Comment:
Can't directly determine where this package was used?

Reply:
in the components repo, but there is is not bundled, so we need to make sure it is available in the client.
I'm sure we'll also need it in the client, for some dynamic html in claire's translations

Reply:
in de components heb ik die dep terug verwijdert, dus hier ook